src/pages/app/menus/AddonMenuItem.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+  import MenuItem from "@/common/MenuItem";
+  import Tooltip from "@/common/Tooltip";
+  import { layout, dispatchAddonSelected,openDispatchAddon } from "@/manager/layout";
+  import {
+    selectedDocsInProject,
+  //   addSelectedDocsToProject,
+  //   removeSelectedDocsFromProject,
+  } from "@/manager/addons";
+  import { handlePlural } from "@/util/string";
+
+  export let addon;
+
+  $: scope = "none"
+  //   $layout.selected.length > 0 ? selectedDocsInProject(addon) : "none";
+
+  $: caption =
```

Comment:
This text should be put in the i18n framework - `handlePlural` should not be used.